src/main/kotlin/org/wycliffeassociates/otter/common/persistence/repositories/ITakeRepository.kt

```diff
@@ -9,4 +9,5 @@ interface ITakeRepository : IRepository<Take> {
     fun insertForContent(take: Take, content: Content): Single<Int>
     fun getByContent(content: Content): Single<List<Take>>
     fun removeNonExistentTakes(): Completable
+    fun markDeleted(obj: Take): Completable
```

Comment:
If the name "obj" is trying to adhere to existing practices, know that I never meant to let "obj" get through PR's.

Let's stick with "take" for the variable name

Reply:
Done.